LoRaEngine/LoraKeysManagerFacade/IoTHubDeviceInfo.cs

```diff
@@ -23,7 +23,15 @@ public DevAddr DevAddr
             }
         }
 
-        public string DevEUI { get; set; }
+        [JsonIgnore]
+        public DevEui? DevEUI { get; set; }
+
+        [JsonProperty("DevEUI")]
+        public string DevEuiString
+        {
+            get => DevEUI?.ToHex();
+            set => DevEUI = string.IsNullOrEmpty(value) ? null : DevEui.Parse(value);
```

Comment:
Converting an _empty string_ to `null` will given inconsistent results:

```c#
var info = new IoTHubDeviceInfo();
info.DevEuiString = string.Empty;
var s = info.DevEuiString; // returns null
````

Reply:
Since `DevEui` needs to be null in some cases, how would you propose to solve this? Use the `default(DevEui)` as a representation of an empty string?

Reply:
A nullable `DevEui`?

Reply:
As discussed offline, removed the check for empty strings entirely with 6ad941ed410bb1b31df520f46b8353801fb70cf8